pkgs/tools/virtualization/linode-cli/default.nix

```diff
@@ -13,9 +13,11 @@
 let
   # specVersion taken from: https://www.linode.com/docs/api/openapi.yaml at `info.version`.
   specVersion = "4.104.0";
+  specSha256 = "0lbky0djqsndp33067q6yir5525m3zcgkpvf1z16mkspkixzadqs";
+  pkgSha256 = "08j7j9fpxh08vkczjqahb8r55bwks8p3ykrv6a2kfi309629nm96";
```

Comment:
If you use update-source-version this can be avoided.

Reply:
I see`update-source-version` looks up for old hash and replaces it. Current solution uses a label and works for any hash. This package is unusual because it has the spec and I would still have to use the label solution for the spec as the word `sha256`, repeats in the file. If it is not a problem, can we leave it as it is?

Reply:
Can we use ``update-source-version`` for the standard fields and only sed for the extra one?

Reply:
I see no advantage in doing so. The source of truth is upstream and I'm getting the data from it already. `update.sh` updates `default.nix` properly without having to lookup any previous hashes (which `update-source-version` requires). The unique label solution used here allow this. I can leave fields blank and run script and have it filled properly.